app/controllers/tanks_controller.rb

```diff
@@ -0,0 +1,11 @@
+class TanksController < ApplicationController
+  before_action :set_tank, only: [:show]
+
+  def show; end
+
+  private
+
+  def set_tank
+    @tank = Tank.find(params[:id])
```

Comment:
After #336 is implemented we will want to follow up and make sure that we scope any queries here to the `current_user`'s organization, but fine to leave as is for now I think